### PR TITLE
AddingBuildScript_ jpam

### PR DESCRIPTION
**Pull request by santosh653**
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ jpam ---
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. bu                                                                                        ild is platform dependent!
[INFO] Compiling 4 source files to /home/ubuntu/jpam/jpam/target/classes
[INFO]
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ jp                                                                                        am ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i                                                                                        .e. build is platform dependent!
[INFO] skip non existing resourceDirectory /home/ubuntu/jpam/jpam/src/test/resou                                                                                        rces
[INFO]
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ jpam --                                                                                        -
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. bu                                                                                        ild is platform dependent!
[INFO] Compiling 7 source files to /home/ubuntu/jpam/jpam/target/test-classes
[WARNING] /home/ubuntu/jpam/jpam/src/test/java/net/sf/jpam/AbstractPamTest.java:                                                                                         /home/ubuntu/jpam/jpam/src/test/java/net/sf/jpam/AbstractPamTest.java uses unch                                                                                        ecked or unsafe operations.
[WARNING] /home/ubuntu/jpam/jpam/src/test/java/net/sf/jpam/AbstractPamTest.java:                                                                                         Recompile with -Xlint:unchecked for details.
[INFO]
[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ jpam ---
[INFO]
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ jpam ---
[INFO] Building jar: /home/ubuntu/jpam/jpam/target/jpam-1.1.jar
[INFO]
[INFO] --- maven-install-plugin:2.4:install (default-install) @ jpam ---
[INFO] Installing /home/ubuntu/jpam/jpam/target/jpam-1.1.jar to /home/ubuntu/.m2                                                                                        /repository/net/sf/jpam/jpam/1.1/jpam-1.1.jar
[INFO] Installing /home/ubuntu/jpam/jpam/pom.xml to /home/ubuntu/.m2/repository/                                                                                        net/sf/jpam/jpam/1.1/jpam-1.1.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.987 s
[INFO] Finished at: 2021-07-20T09:55:51Z
[INFO] ------------------------------------------------------------------------

### CODE REVIEW

**Comment by gururajrkatti** [comment]
There are too many commits here. Can you pl. squash them.

**Comment by santosh653** [comment]
@gururajrkatti Hi Guru, I have squashed all the commits now., Thanks.,

File: j/jpam/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+  Apache License
```
Comment (gururajrkatti) [inline j/jpam/LICENSE:1]: Indentation issue

File: j/jpam/jpam_ubuntu_20.04.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# ----------------------------------------------------------------------------
+#
+# Package           : jpam
+# Version           : master
```
Comment (gururajrkatti) [inline j/jpam/jpam_ubuntu_20.04.sh:4]: Pl. mention what is the version tested

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Comment by santosh653** [comment]
@gururajrkatti Can you please review this?

**Review by gururajrkatti — APPROVED** [review]
ok

**Pull request merged**